<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGUIListConstructor</class>
 <widget class="QDialog" name="CGUIListConstructor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plan Lists editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>List:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="lists_combo"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_warning_color">
       <property name="text">
        <string>Set Warning color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_danger_color">
       <property name="text">
        <string>Set Danger color</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_default_current">
       <property name="text">
        <string>Set line to default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_default_all">
       <property name="text">
        <string>Set all to default</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
